src/composables/useFormSpaceSettings.ts

```diff
@@ -153,6 +156,7 @@ export function useFormSpaceSettings(
       period: formData.voting.period || undefined,
       type: formData.voting.type || undefined,
       quorum: formData.voting?.quorum || undefined,
+      quorumType: formData.voting.quorumType || undefined,
```

Comment:
Shouldn't this default to `default`?
```suggestion
      quorumType: formData.voting.quorumType || 'default',
```

Reply:
As it is pruned on line 96 anyways